mmv1/products/redis/Cluster.yaml

```diff
@@ -63,6 +63,30 @@ examples:
       'deletion_protection_enabled': 'false'
     oics_vars_overrides:
       'deletion_protection_enabled': 'false'
+  - name: "redis_cluster_rdb"
+    primary_resource_id: "cluster-rdb"
+    vars:
+      cluster_name: "rdb-cluster"
+      policy_name: "mypolicy"
+      subnet_name: "mysubnet"
+      network_name: "mynetwork"
+      prevent_destroy: 'true'
+    test_vars_overrides:
+      prevent_destroy: 'false'
+    oics_vars_overrides:
+      prevent_destroy: 'false'
+  - name: "redis_cluster_aof"
```

Comment:
The following error occurs because you haven't set the `deletion_protection_enabled` variable set as part of both rdb and aof tests that you've added.

```hcl
        Error: Invalid expression
        
          on terraform_plugin_test.tf line 16, in resource "google_redis_cluster" "cluster-rdb":
          16:   deletion_protection_enabled = 
        
        Expected the start of an expression, but found an invalid expression token.
```

It's attempting to use a variable that doesn't exist, `  deletion_protection_enabled = {{index $.Vars "deletion_protection_enabled"}}` refer to the other test on how to add the variable. example: https://github.com/GoogleCloudPlatform/magic-modules/blob/c3ac3f99aca02c19bfb629f583d6626264ce0ffb/mmv1/products/redis/Cluster.yaml#L42-L65

Reply:
Ok I've fixed this in the latest rev